src/test/resources/fixtures/hmh/hmh-renlearn/released/caliperEventStarScoreGraded.json

```diff
@@ -21,7 +21,7 @@
     "endedAtTime": "2016-11-15T10:55:12.000Z"
   },
   "eventTime": "2016-11-15T10:57:06.000Z",
-  "edApp": "https://www.hmhco.com/PartnerService",
+  "edApp": "https://www.hmhco.com/RSR",
```

Comment:
hi @VibinD what does RSR stand for?

Reply:
@kevinmcgarry - it means `Renaissance Star Reading` and `Renaissance Star Math`